source/_integrations/modbus.markdown

```diff
@@ -634,6 +634,67 @@ climates:
               description: "Value corresponding to HVAC Heat/Cool mode."
               required: false
               type: [integer, list]
+    fan_mode_register:
+      description: "Configuration of register for Fan mode"
+      required: false
+      type: map
+      keys:
+        address:
+          description: "Address of Fan mode register."
+          required: true
+          type: integer
+        write_registers:
+          description: "Request type, use `write_registers` if true  else `write_register`.
+            If more than one value is specified for a specific mode, only the first one is used for writing to the register."
+          required: false
+          type: boolean
+          default: false
+        values:
+          description: "Mapping between the register values and Fan modes
+            This is typically used to control one of: Speed, Direction or On/Off state."
+          required: true
+          type: map
+          keys:
+            state_fan_on:
```

Comment:
It would be normal to have a default value.

Reply:
Again, I was copying the setup for HVAC mode here. I may be mistaken, but if I had default values here, wouldn't that cause the component to assume that the mode was supported, since lack-of-value is interpreted as unsupported mode during setup.

I could add a comment to this effect for both modes?